apps/indexer/lib/indexer/fetcher/on_demand/token_balance.ex

```diff
@@ -195,9 +195,16 @@ defmodule Indexer.Fetcher.OnDemand.TokenBalance do
   end
 
   defp prepare_updated_balance({{:error, error}, ctb}, block_number) do
+    error_message =
+      if ctb.id do
```

Comment:
Agree with bot, shouldn't it be
```suggestion
      if ctb.token_id do
```
?

Reply:
Sure, fixed in https://github.com/blockscout/blockscout/pull/11524/commits/79098a7de81d54112ae3c69e9e5944629e78033c